For whatever reasons linking against ppui pulls in osinterface as
dependency as well. Having 'osinterface.a' specified twice leads
to multiple definitions errors at the linking step.

As a quick-fix remove the explicit dependency and only reference
the headerfiles.
--- src/src/tracker/CMakeLists.txt
+++ src/src/tracker/CMakeLists.txt
@@ -204,10 +204,19 @@
         ${PROJECT_BINARY_DIR}/src/tracker
 )
 
+target_include_directories(tracker
+    PRIVATE
+        ${CMAKE_CURRENT_SOURCE_DIR}/../ppui/osinterface
+)
+
+target_include_directories(tracker
+    PRIVATE
+        ${CMAKE_CURRENT_SOURCE_DIR}/../ppui/osinterface/posix
+)
+
 target_link_libraries(tracker
     fx
     milkyplay
-    osinterface
     ppui
 )
 
